pkg/deploy/predeploy.go

```diff
@@ -24,6 +25,8 @@ import (
 	"github.com/Azure/ARO-RP/pkg/util/keyvault"
 )
 
+const rotateSecretAfter = time.Hour * 72
```

Comment:
Do I understnad correctly that we want to rotate on each deployment? But if we had a deployment in last 72 hours - we do not rotate?

Might be worth documenting it somehow.

Reply:
Yes that is correct.  In case we fail a deploy we don't keep creating new secret versions every time it fails.  

I will find a place to document.

Reply:
Having a comment above the constant might be good enough

Reply:
Updated.  

```
// Rotate the secret on every deploy of the RP iff the most recent
// secret is less than 3 days old
const rotateSecretAfter = time.Hour * 72